testutil/filepresencechecker.go

```diff
@@ -28,18 +28,35 @@ import (
 
 type filePresenceChecker struct {
 	*check.CheckerInfo
-	present bool
+	statFunc func(name string) (os.FileInfo, error)
+	present  bool
 }
 
-// FilePresent verifies that the given file exists.
+// FilePresent verifies that the given file exists, following symlinks.
 var FilePresent check.Checker = &filePresenceChecker{
 	CheckerInfo: &check.CheckerInfo{Name: "FilePresent", Params: []string{"filename"}},
+	statFunc:    os.Stat,
 	present:     true,
 }
 
-// FileAbsent verifies that the given file does not exist.
+// FilePresent verifies that the given file/symlink exists.
+var FileLpresent check.Checker = &filePresenceChecker{
```

Comment:
nitpick: my preference would be to call this LFilePresent etc

Reply:
Changed